scripts/pi-hole/js/groups-domains.js

```diff
@@ -55,10 +55,72 @@ $(function () {
 
   $("#add2black, #add2white").on("click", addDomain);
 
+  $("#new_domain").on("paste", showSuggestDomains);
+
   utils.setBsSelectDefaults();
   getGroups();
 });
 
+function showSuggestDomains(event) {
+  function createButton(hostname) {
+    // Purposefully omit 'btn' class to save space on padding
+    return $('<button type="button" class="btn-link btn-block text-right">')
+      .append($("<i>").text(hostname))
+      .click(function () {
+        hideSuggestDomains();
+        newDomainEl.val(hostname);
+      });
+  }
+
+  var newDomainEl = $("#new_domain");
+  var suggestDomainEl = $("#suggest_domains");
+  var clipboardData = event.originalEvent.clipboardData.getData("text");
+
+  // Remove input listener otherwise the paste event will trigger a hide action
+  newDomainEl.off("input", hideSuggestDomains);
+
+  // setTimout is needed to get the pasted value from #new_domain
+  setTimeout(function () {
+    try {
+      // Only activate if the pasted text is same as #new_domain value
+      if (clipboardData !== newDomainEl.val()) {
+        hideSuggestDomains();
+        return;
+      }
+
+      // URL is not supported in IE11, but we are in a try-block so we can ignore it
+      // eslint-disable-next-line compat/compat
```

Comment:
Meanwhile, we dropped IE11 support (https://github.com/pi-hole/AdminLTE/pull/2210). I think you need to rebase on `devel` and remove the line

```suggestion

```